components/landing/TheHeroMoment/index.vue

```diff
@@ -46,20 +51,17 @@ export default class TheHeroMoment extends Vue {
 .the-hero-moment {
   position: relative;
   overflow: hidden;
-  margin-bottom: $layout-05;
-  // In Figma, the height is not enforced but the background is always
-  // visible completely so we do it in the CSS. A small correction is needed
-  // to be able of displaying the bottom lines of the grid.
-  height: calc(56rem + 2px);
+  margin-bottom: $spacing-07;
```

Comment:
When it's possible, we are trying to use `$layout` variables instead of `$spacing` on the "margin" properties. Would you mind to replace this `$spacing-07` with `$layout-03` that has the same value?

Reply:
As discussed offline, we'll be replacing all `$layout` variables with `$spacing` variables with https://github.com/Qiskit/qiskit.org/issues/1789.